src/test/ui/const-generics/issues/issue-72819-generic-in-const-eval.rs

```diff
@@ -0,0 +1,17 @@
+#![feature(const_generics)]
+#![allow(incomplete_features)]
+struct Arr<const N: usize>
```

Comment:
can you add a brief comment explaining the role of this test?

Reply:
Just saying "regression test for issue XXX" probably suffices, although if you can provide any more details of what the ICE was about, that's sometimes useful